mmpose/models/losses/classification_loss.py

```diff
@@ -153,3 +153,41 @@ def forward(self, pred_simcc, gt_simcc, target_weight):
                 self.criterion(coord_y_pred, coord_y_gt).mul(weight).sum())
 
         return loss / num_joints
+
+
+@MODELS.register_module()
+class InfoNCELoss(nn.Module):
+    """InfoNCE loss for training a discriminative representation space with a
+    contrastive manner.
```

Comment:
Could you please add the paper reference for InforNCELoss and FocalHeatmapLoss?

Reply:
Sure. The references have been added